app/controllers/auctions_controller.rb

```diff
@@ -32,6 +32,14 @@ def update
 
   private
 
+  def find_auction
+    if params[:token]
+      Auction.find_by(token: params[:token])
```

Comment:
I'm wondering if we should name this query, and put it in `AuctionQuery` similiar to https://github.com/18F/micropurchase/blob/develop/app/models/auction_query.rb#L12-L14